k6/health.js

```diff
@@ -0,0 +1,10 @@
+import http from 'k6/http';
+import { sleep } from 'k6';
```

Comment:
Codacy has a fix for the issue: [Replace `'k6'` with `"k6"`](https://app.codacy.com/gh/dasch-swiss/dsp-api/pullRequest?prid=10006168)
```suggestion
import { sleep } from "k6";
```